openprescribing/gcutils/bigquery.py

```diff
@@ -110,6 +116,14 @@ def query(self, sql, legacy=False, **options):
 
         return query
 
+    def query_into_dataframe(self, sql, legacy=False):
+        kwargs = {
+            'project_id': self.project_name,
+            'verbose': False,
+            'dialect': 'legacy' if legacy else 'standard',
+        }
+        return pd.read_gbq(sql, **kwargs)
```

Comment:
The catch-then-add-line-numbers logic in the version you removed has been a very useful thing for me in the past.